source/conf.py

```diff
@@ -76,3 +93,10 @@
 numfig = True
 # Necessary setting for sphinxcontrib-bibtex >= 2.0.0
 bibtex_bibfiles = ['bibliography.bib']
+
+variables_to_export = [
+    "version",
+]
+frozen_locals = dict(locals())
+rst_epilog = '\n'.join(map(lambda x: f".. |{x}| replace:: {frozen_locals[x]}", variables_to_export))
```

Comment:
Please add a comment about where/how this value is used by Sphinx.

Reply:
Added in https://github.com/openqasm/openqasm/pull/452/commits/517cbc386d5d70ebce6ebbce1e7a419e91e65dee